web/modules/custom/weather_block/src/Plugin/Block/WeatherBlock.php

```diff
@@ -130,19 +138,19 @@ public function blockForm($form, FormStateInterface $form_state) {
       '#collapsed' => FALSE,
     ];
 
-    $form['city_selection']['cities'] = [
+    $form['city_selection']['city'] = [
```

Comment:
This now seems to be a default city.
But naming doesn't indicate that in any way